lighthouse-cli/bin.ts

```diff
@@ -62,7 +62,8 @@ const cliFlags = yargs
     'list-trace-categories',
     'config-path',
     'perf',
-    'port'
+    'port',
+    'timeout'
```

Comment:
`timeout` makes me think of a timeout for the whole run or something. Maybe some non-awkward version of `page-load-timeout`?